stake-pool/js/package.json

```diff
@@ -29,6 +27,7 @@
     "access": "public"
   },
   "browser": "dist.browser/index.js",
+  "module": "dist.browser/index.esm.js",
   "main": "dist/index.js",
```

Comment:
Would it be possible to change this to `dist.browser/index.cjs.js` for compatibility with CommonJS?

Reply:
better now?

Reply:
Yes, works perfectly, thank you!